refactor.rb

```diff
@@ -0,0 +1,15 @@
+class People
+	attr_reader :age, :phase
+	attr_accessor :name
+
+	def initialize(options={})
+		@phase = 3 # not sure 
+		@age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+	end
+
+	def offer_high_five   #not sure if this is appropiate cause it's a behavior
+    "High five!"
+  end
+
```

Comment:
The code here is good, but the formatting is not.  Very difficult to read.  Make sure that the body of a block is indented one tab further than the enclosing opener ` ... end` (e.g. `def ... end` or `class ... end` or `do ... end`)